ts/features/itwallet/navigation/routes.ts

```diff
@@ -12,7 +12,10 @@ export const ITW_ROUTES = {
     ALREADY_ACTIVE_SCREEN: "ITW_ALREADY_ACTIVE_SCREEN"
   } as const,
   IDENTIFICATION: {
-    MODE_SELECTION: "ITW_IDENTIFICATION_MODE_SELECTION",
+    LEVEl_SELECTION: {
+      L2: "ITW_IDENTIFICATION_LEVEL_SELECTION_L2",
+      L3: "ITW_IDENTIFICATION_LEVEL_SELECTION_L3"
+    },
```

Comment:
It is a name that better fit the names screen. What do you think?
```suggestion
    MODE_SELECTION: {
      L2: "ITW_IDENTIFICATION_MODE_SELECTION_L2",
      L3: "ITW_IDENTIFICATION_MODE_SELECTION_L3"
    },
```